app/javascript/components/public_health/Import.js

```diff
@@ -225,7 +225,7 @@ class Import extends React.Component {
               </Button>
             )}
             {this.state.phased.length > 0 && (
-              <ProgressBar animated striped className="my-3" now={Math.round((this.state.progress + 1 / this.state.phased.length) * 100)} />
+              <ProgressBar animated striped className="my-3" now={((this.state.progress + 1) / this.state.phased.length) * 100} />
```

Comment:
Not sure if I'm missing something but I'm still seeing it go to 100 immediately on a large import.

https://user-images.githubusercontent.com/8235974/134074394-13a3cadb-17eb-4423-8acf-ace88ae17a49.mov

Reply:
Ah sorry, I should've been more clear and added some screenshots (or a lovely video like you did). The progress bar this is meant to fix is the one that pops up in the next modal after the one you showed, once you hit "Import All".